docs/concepts/html/components.md

```diff
@@ -99,3 +112,34 @@ impl Component for List {
 }
 ```
 
+## Transformers
+
+Whenever you set a prop its value goes through a transformation step first.
+If the value already has the correct type, this step doesn't do anything.
+However, transformers can be useful to reduce code repetition.
+
+The following is a list of transformers you should know about:
+
+- `&T` -> `T`
+
+  Clones the reference to get an owned value.
+
+- `&str` -> `String`
+
+  Allows you to use string literals without adding `.to_owned()` at the end.
+
+- `T` -> `Option<T>`
+
+  Wraps the value in `Some`.
+
+```rust
+struct Props {
+    unique_id: Option<usize>,
+    text: String,
+}
+
+// transformers allow you to write this:
+yew::props!(Props { unique_id: 5, text: "literals are fun" });
+// instead of:
+yew::props!(Props { unique_id: Some(5), text: "literals are fun".to_owned() });
```

Comment:
I think this no longer works as a demonstration of transformers because the syntax is so far removed.
Wouldn't want to confuse readers into thinking that it doesn't apply to the `html!` macro.